cfme/fixtures/pytest_selenium.py

```diff
@@ -325,6 +325,8 @@ def _nothing_in_flight():
         anything_in_flight |= running["prototype"] > 0
         anything_in_flight |= running["spinner"]
         anything_in_flight |= running["document"] != "complete"
+        anything_in_flight |= running["autofocus"] > 0
+        anything_in_flight |= running["miqQE"] > 0
```

Comment:
If we are checking all with this......do we need the specific autofocus from above?